src/Symfony/Component/Messenger/Tests/Stamp/RedeliveryStampTest.php

```diff
@@ -28,4 +28,15 @@ public function testSerialization()
         $stamp = new RedeliveryStamp(10, null, null, \DateTimeImmutable::createFromFormat(\DateTimeInterface::ISO8601, '2005-08-15T15:52:01+0000'));
```

Comment:
we should update this also, isn't it?

Reply:
It is not strictly required. I am doing that in [a PR to 6.0](https://github.com/symfony/symfony/pull/41319/files#diff-ebc86daca4f203e7fd0c91af5b6d4bc51dd2fe39c57ea4bb1d2214d7ef39f9c6L28)

Reply:
doing it here might help reduce merge conflicts